src/main/features/response/views/when-will-you-pay.njk

```diff
@@ -0,0 +1,47 @@
+{% extends "layout.njk" %}
+
+{% set title = 'Money claim' %}
+
+{% from "form.njk" import saveAndContinueButton, csrfProtection, radioGroup, radioGroupWithHintPanel, errorSummary %}
+
+{% block content %}
+  <div class="column-two-thirds">
+
+    <h1 class="heading-large">
+      {{ t('When will you pay?') }}
```

Comment:
`{% set heading = 'Explain why you can’t pay now' %}` before the content block instead?